components/backup-stream/src/endpoint.rs

```diff
@@ -429,11 +430,13 @@ where
         // Let L be the instant some key locked, U be the instant it unlocked,
         // +---------*-------L-----------U--*-------------+
         //           ^   ^----(1)----^      ^ We get the snapshot for initial scanning at here.
-        //           +- If we issue refresh resolver at here, and the cmd batch (1) is the last cmd batch of the first observing.
-        //              ...the background initial scanning may keep running, and the lock would be sent to the scanning.
-        //              ...note that (1) is the last cmd batch of first observing, so the unlock event would never be sent to us.
-        //              ...then the lock would get an eternal life in the resolver :|
-        //                 (Before we refreshing the resolver for this region again)
+        //           +- If we issue refresh resolver at here, and the cmd batch (1) is the last cmd
```

Comment:
Also bad format.

Reply:
should use `-` to indicate list item, will fix